packages/react/src/primitives/PasswordField/ShowPasswordButton.tsx

```diff
@@ -0,0 +1,24 @@
+import * as React from 'react';
+import { ShowPasswordButtonProps } from '../types/';
+import { Button } from '../Button';
+import { ComponentClassNames } from '../shared/constants';
+import { IconVisibility } from '../Icon';
+import { Text } from '../Text';
+import classNames from 'classnames';
+
+export const ShowPasswordButton: React.FC<ShowPasswordButtonProps> = ({
+  onClick = () => {},
+}) => {
+  return (
+    <Button
+      className={classNames(
+        ComponentClassNames.FieldShowPassword,
+        ComponentClassNames.FieldGroupField
+      )}
+      onClick={onClick}
+    >
+      <Text className="sr-only">Show password</Text>
```

Comment:
What is the benefit from having screen reader only text and an empty alt tag over having this in an alt tag?

Reply:
Good call. I think I should actually move "Show password" to the Button as an `aria-label`, and should toggle it between "hide password" and "show password"

Reply:
https://gomakethings.com/icon-accessibility-and-aria-label/